doc/content/devices/adding-devices/_index.md

```diff
@@ -14,41 +14,99 @@ This section contains instructions for adding devices in {{% tts %}}.
+Devices are managed under applications. An application can contain an unlimited number of devices, but it can be helpful to sort devices in to applications by function or geographical area, to make the integrations and live data views more useful.
 
-{{< note "The PHY version represents the revision of the LoRAWAN Regional Parameters that the device expects, and must be correlated with the MAC version." />}}
+To create a device, first open the application you wish to add the device in. Go to **End devices** in the left menu and click on **+ Add end device** to reach the end device registration page.
 
-{{< figure src="device-creation-1.png" alt="Creating a new device" >}}
+{{< figure src="application-overview.png" alt="Application overview" >}}
 
-### Over-The-Air-Activation (OTAA) Device
+### Using the LoRaWAN Device Repository
 
-After filling the fields in the **General Settings** section, scroll to the lower part of the device registration page and make sure that **Over The Air Activation** is selected. 
+The [LoRaWAN device repository](https://github.com/TheThingsNetwork/lorawan-devices) contains device profiles, LoRaWAN information, codecs, and more, for many LoRaWAN devices. Using the device repository to add devices in {{% tts %}} automatically uses the correct LoRaWAN version and regional parameters version, which means less information for you to find!
 
-Fill the **JoinEUI** (AppEUI in LoRaWAN versions before 1.1), **DevEUI** and **AppKey**. The **NwkKey** is only needed for LoRaWAN version 1.1 or later. All other fields on the page are optional. 
+To use the device repository, make sure the **From the LoRaWAN Device Repository** button is selected. Then, select the **Brand**, **Model**, **Hardware Version**, **Software Version**, and **Region** for your device.
 
-Press **Create Device** to create the device.
+{{< note "If your device is not in the device repository, see [Manually Create End Device](#manually-create-end-device) below." />}}
 
-{{< figure src="device-creation-otaa.png" alt="Creating an OTAA device" >}}
+{{< figure src="device-repo.png" alt="Creating a new device with the Device Repository" >}}
 
-You will now reach the device overview page for your device. The end device should now be able to join the private network.
+Choose a **Frequency plan** appropriate for your region. Your device and gateway must use the same frequency plan to communicate.
 
-{{< note >}} If you do not have a `JoinEUI` or `AppEUI`, it is okay to use `0000000000000000`. Be sure to use the same `JoinEUI` in your device as you enter in {{% tts %}}. {{</ note >}}
+Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use all 0's, and then program the same JoinEUI/AppEUI in the device.
 
-{{< figure src="device-otaa-created.png" alt="OTAA device overview" >}}
+Enter your **DevEUI**.
```

Comment:
Maybe add a similar line as for the JoinEUI/AppEUI, because DevEUI can be provided by the manufacturer, but I think we can issue DevEUIs now too

Reply:
I don't know what the official word is on this, so going to leave it for now.